package.json

```diff
@@ -51,5 +51,8 @@
     "lodash": "^4.17.21",
     "minimatch": "^5.1.0",
     "table": "^6.8.0"
+  },
+  "peerDependencies": {
+    "jest": ">=28.1.1"
```

Comment:
Picked this as it's when `@jest/types` was introduced.

https://github.com/jestjs/jest/releases/tag/v28.1.1